code/game/objects/items/devices/multitool.dm

```diff
@@ -76,7 +75,8 @@
 	track_delay = 5
 
 /obj/item/device/multitool/ai_detect/admin/multitool_detect()
-	for(var/mob/J in range(rangewarning,src))
+	var/turf/our_turf = get_turf(src)
+	for(var/mob/J in range(rangewarning,get_turf(src)))

```

Comment:
wait uhhh declaring new vars in the detection and not in the multitool? :p 
sorry still very very new and reading already coded things to figure out how to make things